common/include/Pcsx2Defs.h

```diff
@@ -168,7 +168,7 @@ static const int __pagesize = PCSX2_PAGESIZE;
 #define PCSX2_ALIGNED16_EXTERN(x) extern __declspec(align(16)) x
 
 #define __noinline __declspec(noinline)
-#define __threadlocal __declspec(thread)
+#define thread_local __declspec(thread)
```

Comment:
@KousukeItsagame 
In case you don't know how define work. Otherwise you can skip this comment.
```
#define word1 word2
```
Mean to the compile to replace occurance of word1 by word2. In the previous code the virtual keyword `__threadlocal` was replaced by the compiler extension `__declspec(thread)`. So it allow to use
```
#define __threadlocal __declspec(thread)
__threadlocal my_var;
```
and the compiler see it as
```
__declspec(thread) my_var;
```
 As you notice C++11 uses a standard name `thread_local` so we don't need anymore the intermediate keyword. You can now use it directly. Hence
```
thread_local my_var;
```